VENDORING.md

```diff
@@ -0,0 +1,238 @@
+# Perform the vendoring of the Kata Containers project
```

Comment:
"Performing vendoring for the Kata Containers project" ?

Reply:
done